pro/src/pages/Home/OffererStats/OffererStats.tsx

```diff
@@ -3,12 +3,12 @@ import { useLocation } from 'react-router-dom'
 
 import { Events } from 'core/FirebaseEvents/constants'
 import useAnalytics from 'hooks/useAnalytics'
-import { ReactComponent as ShadowCalendar } from 'icons/shadow-calendar.svg'
-import { ReactComponent as ShadowTropheeIcon } from 'icons/shadow-trophee.svg'
-import { ReactComponent as StrokeEuro } from 'icons/stroke-euro.svg'
+import fullShowIcon from 'icons/full-show.svg'
+import shadowCalendarIcon from 'icons/shadow-calendar.svg'
+import shadowEuroIcon from 'icons/shadow-euro.svg'
+import shadowTropheeIcon from 'icons/shadow-trophee.svg'
```

Comment:
Est-ce que on ne le renommerait pas en shadow**Trophy**Icon ?
```suggestion
import shadowTrophyIcon from 'icons/shadow-trophy.svg'
```

Reply:
Si !